.github/workflows/Unittesting.yml

```diff
@@ -6,16 +6,14 @@ jobs: # what should we do when it is run
   build:
     runs-on: ubuntu-latest # platform it runs on
     steps:
-    - uses: actions/checkout@v4 # checkout your code
+    - uses: actions/checkout@v2 # checkout your code
     - name: Set up Python
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@v2
       with:
         python-version: '3.10' # specify python version
     - name: Install dependencies # which libraries do we need for testing
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt # they should be listed in this file
     - name: Test with pytest
-      run: |
-        pip install pytest
-        pytest tests.py
+      run: pytest test_matrix_multiplication.py
```

Comment:
You’re added line will work, but removing the run statement likely will make it fail

Reply:
Yeah professor, I tried this.